### ISSUE TITLE
folder i18n needed in example website

### ISSUE THREAD

**Issue by moreedri** [theNewDynamic/gohugo-theme-ananke#383]
If the i18n files remain under the `/theme` folder, live previewing in hugo will render translated strings, but the rendered website in `/public` will not.

The example website included with the theme should also contain the i18n folder and the `en.toml` language file (as the example site is in english)

**Comment by regisphilibert** [comment]
I'm going to need more context here... 

The string are rendered fine through the exampleSite (which is not multilingual) on my local machine. I can also confirm that the strings gets rendered in their matching language on a multilingual project.

As for your suggestion, the i18n directory is picked up from the theme if not found on the project level, so I fail to grasp the problem with the directory location.

Then again, this issue description is light so I might be missing something...

**Comment by littleninja** [comment]
My limited understanding of the difference between i18n and multilingual:

- i18n: Tokenize and replace site chrome (ex. "read more" button)
- multilingual: Configure site to route multiple languages and serve translations of `/content`

Again, I'm new to Hugo and [this is the doc I'm reading on multilingual support](https://gohugo.io/content-management/multilingual). Under ["Configure Multilingual Multihost"](https://gohugo.io/content-management/multilingual/#configure-multilingual-multihost), I noted that the sample given publishes to:

```
public
 |_ en
 |_ fr
```

If the suggestion is multilingual support including i18n, did #386 help address this?

**Comment by regisphilibert** [comment]
> If the suggestion is multilingual support including i18n, did #386 help address this?

Yes for me it did. For better understanding you can check the blog post referenced in the doc.

**Comment by davidsneighbour** [comment]
Closing this for now. i18n for the theme will be refactored/fixed in a future release. Sample websites should not require i18n in addition to that.
